src/main/java/io/debezium/connector/yugabytedb/YugabyteDBConnectorConfig.java

```diff
@@ -738,7 +799,7 @@ public enum AutoCreateMode implements EnumeratedValue {
         /**
          * Enable publication for all tables.
          */
-        ALL_TABLES("all_tables"),
+        ALL_TABLES("all tables"),
```

Comment:
Why are we changing this?

It is `all_tables` in the upstream connector: https://github.com/debezium/debezium/blob/616bb52f208dd5e0aad8d95b4091bedf2fd5ef51/debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java#L556